docassemble/StudentEvaluations/data/questions/massachusetts_educational_evaluations_request.yml

```diff
@@ -653,6 +653,16 @@ code: |
   if task_not_yet_performed('10 day reminder text') and date_difference(starting=letter_created_date).days > 10 and wants_reminders:
     send_sms(to=requestor, body='Have you heard from the school yet?', task='10 day reminder text')
   response()
+#event: cron_daily(10,20)
```

Comment:
this is commented out code

Reply:
That was a test case to see if the email works